Plotting.md

```diff
@@ -0,0 +1,8 @@
+## Plots and Charts
+
+ * Creating plots with Matplotlib
+ * Maps and projections with Cartopy
+ * Plotting stats with Seaborn
+ * Importing data into GIS
```

Comment:
Is that `into GIS` or `from GIS`?

Reply:
into gis is what i had, in case someone wanted to do fancy mapping